tests/fw_rules_tests/policies/expected_output/calico-testcase13-scheme_output.txt

```diff
@@ -1,3 +1,3 @@
 final fw rules for query: open-default-TCP-test, config: open-default-TCP:
-src_ns: [default] src_pods: [*] dst_ns: [default] dst_pods: [*] conn: All but UDPLite
+src_ns: [default] src_pods: [*] dst_ns: [default] dst_pods: [*] conn: All but MobilityHeader
```

Comment:
This doesn't make sense. The NetworkPolicies refer to `UDPLite` (as string). See `tests/calico_testcases/example_policies/testcase13-protocol/testcase13-also-with-deny.yaml`.
How do we get `MobilityHeader`? Something must be broken somewhere.

Reply:
Test fixed